### PR TITLE
Remove Java

### PR DESCRIPTION
**Pull request by zrfisaac**
Removal of Java Coffee Cup Logo
**Issue:** #7374

Brand Name
Java

Reason for Removal
Emails received at [removals@simpleicons.org](mailto:removals@simpleicons.org) from an Oracle Corporation attorney.

😕

### CODE REVIEW

**Comment by sachinraja** [comment]
Thanks! You'll also need to remove our use of `assets/readme/java-white.svg` from the README. 

Marking this as draft so it doesn't accidently get merged before the next major.

**Comment by zrfisaac** [comment]
Year, I saw that after, I've Already made this, that for the advice! 🙌

**Comment by zrfisaac** [comment]
I should remove it from here too? because it is not going to load anymore
Capturar

**Comment by mondeja** [comment]
> I should remove it from here too? because it is not going to load anymore

Yes, but I think that the best aproach would be to add an empty icon, so the `Java Library` link is not misaligned with the rest of the third party libraries. Would you mind to replace it by a 24x24 empty icon @zrfisaac?

**Comment by sachinraja** [comment]
Yes, that's what I was referring to by this:
> You'll also need to remove our use of assets/readme/java-white.svg from the README.

**Comment by zrfisaac** [comment]
I've removed like you said like you can see bellow :
image

But I had one idea, why not use the [OpenJDK](https://openjdk.java.net) logo !? 

_OpenJDK (Open Java Development Kit) is a free and open-source implementation of the Java Platform, Standard Edition (Java SE)._

I'd like to recomend use the OpenJDK logo. what do you think @sachinraja ?
openjdk

**Comment by sachinraja** [comment]
Great idea! Let's do that.

**Comment by zrfisaac** [comment]
Here is the pull request #7391 with this logo.

**Comment by zrfisaac** [comment]
Alright, after the merger of the OpenJDK logo, I will do the change.

**Comment by zrfisaac** [comment]
I've made the change, it's how it's going to look like.

image

**Comment by sachinraja** [comment]
@zrfisaac we can make the readme changes now, so could you make a separate PR for that?

File: assets/readme/openjdk-white.svg [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (mondeja) [inline assets/readme/openjdk-white.svg:1]: Hi @zrfisaac 

You need to use the same content of `openjdk.svg` but adding `fill="white"` attribute or [the README test will not pass](https://github.com/simple-icons/simple-icons/runs/6340479990?check_suite_focus=true). See the other icons in  [assets/readme](https://github.com/simple-icons/simple-icons/tree/b2cf3885ce82351a74b3562c7a19b9e7a45ce67b/assets/readme) directory as reference. Thanks!

**Review by mondeja — CHANGES_REQUESTED** [review]

File: assets/readme/openjdk-white.svg (hunk 1 above)
Comment (zrfisaac) [reply assets/readme/openjdk-white.svg:1]: Thanks I fogot to do that, I will fix today!

Comment (mondeja) [reply assets/readme/openjdk-white.svg:1]: But please, separate the logo change in the third party plugin from this PR so we can update it without a major version release, see https://github.com/simple-icons/simple-icons/pull/7378#issuecomment-1120407819

**Review by mondeja — APPROVED** [review]
I've continued with the work here. This is ready, thanks for your work @zrfisaac 👍🏼

**Review by sachinraja — APPROVED** [review]

**Pull request merged**